.jshintrc

```diff
@@ -59,7 +59,7 @@
     "shadow"        : false,       // Suppresses warnings about variable shadowing i.e. declaring a variable that had been already declared somewhere in the outer scope.
     "sub"           : false,       // Suppresses warnings about using [] notation when it can be expressed in dot notation.
     "supernew"      : false,       // Suppresses warnings about "weird" constructions like new function () { ... } and new Object;.
-    "validthis"     : true,       // Suppresses warnings about possible strict violations when the code is running in strict mode and you use this in a non-constructor function.
+    "validthis"     : true,        // Suppresses warnings about possible strict violations when the code is running in strict mode and you use this in a non-constructor function.
```

Comment:
Oh I see. Was comparing it to the red negative diff line instead of the one above :P